### ISSUE TITLE
[Feature Request] Show Chapter Released Date in Manga Info

### ISSUE THREAD

**Issue by Von07** [fmd-project-team/FMD#120]
I just wondering if its possible to show the released date of the chapters in manga info.. I know that it will takes alot of time. Just tell me if its not possible as of moment.. Thank you

Ex. 
Mangarock - [Chapter]- [Date]
![Request1](https://user-images.githubusercontent.com/21298898/56462739-e49d5200-63fa-11e9-957d-e3733d98a342.png)

**Comment by azraei97** [comment]
I also agree with this. Sometimes website update the chapter without change the name. So only with chapter date we know it new or old
